src/pages/questions/ProductInformation.jsx

```diff
@@ -130,6 +150,19 @@ const ProductInformation = ({ question }) => {
                   style={{ maxWidth: 300, maxHeight: 300 }}
                 />
               </Zoom>
+              {flagged[src[src.length - 5]] ? (
+                <FlagIcon
+                  onClick={() =>
+                    deleteFlagImage(src, question.barcode, src[src.length - 5])
+                  }
+                />
+              ) : (
+                <OutlinedFlagIcon
+                  onClick={() =>
```

Comment:
better to use an [IconButton](https://mui.com/material-ui/api/icon-button/#main-content) and associate the `onClick` to the button